hypha/static_src/src/javascript/apply/tinymce-word-count.js

```diff
@@ -10,17 +10,23 @@
 
     const observer = new MutationObserver(function (mutations) {
         mutations.forEach((mutation) => {
+            const mutation_selector = '#' + mutation.target.id;
             const word_count = mutation.target.innerText.match(/\d+/)[0];
-            const word_count_node = document.querySelector('#' + mutation.target.id);
-            if (word_count <= 800) {
+            const word_count_node = document.querySelector(mutation_selector);
+            const word_limit = $(mutation_selector).parents().eq(5).attr('data-word-limit');
+            const percent_to_get = 20;
+            const word_limit_to_show_warning = word_limit - (percent_to_get / 100) * word_limit;
+
+            if (word_count <= word_limit_to_show_warning) {
+                word_count_node.setAttribute('data-after-word-count', ' out of ' + word_limit);
```

Comment:
@frjo 

> Maybe the word counter at the bottom right can show "0/1000 WORDS" or some better way?

I have set the first one, 

**1.**

![Free Fund (3)](https://user-images.githubusercontent.com/6290791/79547905-3df19d00-80b2-11ea-85b7-1661607ead83.png)

**2. With warning message**

![Free Fund (4)](https://user-images.githubusercontent.com/6290791/79547926-49dd5f00-80b2-11ea-803c-a331fbdb8c03.png)

3.

 ![Free Fund (5)](https://user-images.githubusercontent.com/6290791/79547943-519d0380-80b2-11ea-9460-b35bf9b6be58.png)

4.

 ![Free Fund (6)](https://user-images.githubusercontent.com/6290791/79547965-58c41180-80b2-11ea-8298-3c09b3f0fa25.png)

I have tried `0/1000 WORDS` but tinymce already setting word count and doing DOM manipulation, so when I trying to set this, it was freezing the browser.